### PR TITLE
Add Sticky Footer

### LINKED ISSUE #8: Design podcast list page
No pagination is required at this moment. Just order them by creation date

### PR DESCRIPTION
**Pull request by maryamyasaei**
resolve #8 issue

### CODE REVIEW

File: templates/base/index.html [hunk 1]
```diff
@@ -15,6 +15,7 @@
     <link rel="stylesheet" href="{% static 'css/style.css' %}">
 </head>
 {%  include './navbar.html' %}
+{%  include './footer.html' %}
```
Comment (hesamd) [inline templates/base/index.html]: Please replace this to the line after `{% block content %}This is the content area{% endblock %}`

File: templates/base/navbar.html [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
-  <div class="collapse navbar-collapse" id="navbarSupportedContent">
-    <ul class="navbar-nav mr-auto">
-      <li class="nav-item active">
-        <a class="nav-link" href="/">Home <span class="sr-only">(current)</span></a>
-      </li>
-    </ul>
-  </div>
-</nav>
+
+     <nav class="navbar navbar-expand-lg navbar-dark  bg-dark">
```
Comment (hesamd) [inline templates/base/navbar.html]: Your change in one branch, shouldn't include the other changes in another branch

**Review by hesamd — CHANGES_REQUESTED** [review]

File: templates/base/index.html (hunk 1 above)
Comment (maryamyasaei) [reply templates/base/index.html]: done

**Review by hesamd — APPROVED** [review]

File: templates/base/footer.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<footer class="container-fluid">
+            <footer class="container-fluid">
```
Comment (hesamd) [inline templates/base/footer.html]: You have two footers in your commit

Comment (hesamd) [inline templates/base/footer.html]: you have two footers in html, also don't forget to add a new line in the end and fix the indentation

**Review by hesamd — CHANGES_REQUESTED** [review]

File: templates/base/footer.html (hunk 3 above)
Comment (maryamyasaei) [reply templates/base/footer.html]: done!!!

Comment (hesamd) [reply templates/base/footer.html]: It's not done, your indentation is not fixed in this file and the other file doesn't have a new line in the end.

Comment (maryamyasaei) [reply templates/base/footer.html]: screenshot from 2018-07-05 22-01-06
how is it now?

Comment (hesamd) [reply templates/base/footer.html]: looks good, it's not pushed yet though. You need to fix index.html as well.

**Review by hesamd — APPROVED** [review]

**Pull request merged**